EIPS/eip-draft_hfmeta_metropolis.md

```diff
@@ -0,0 +1,35 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Hardfork Meta: Metropolis
+    Author: Alex Beregszaszi
+    Type: Standard Track
+    Category: Core
+    Status: Draft
+    Created: 23/04/2017
+    Requires: 86, 96, 98, 100, 140, 196, 197, 198, 211, 214
+
+## Abstract
+
+This specifies the changes included in the hard fork named Metropolis.
+
+## Specification
+
+- Codename: Metropolis
+- Activation:
+  - Block not specified yet
+- Included EIPs:
+  - EIP 86 (Abstraction of transaction origin and signature)
+  - EIP 96 (Blockhash refactoring)
+  - EIP 98 (Removal of intermediate state roots from receipts)
```

Comment:
EIP 98 also has different options.  In the last coredev meeting, we chose option 1.